manifests/c/CrystalArtSoftware/NewsLeecher/7.0.0.0/CrystalArtSoftware.NewsLeecher.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: CrystalArtSoftware.NewsLeecher
+PackageVersion: 7.0.0.0

```

```suggestion
PackageVersion: v7.0 Final
```
This must match Control Panel to prevent upgrade loops.